### ISSUE TITLE
Not allowed

### ISSUE THREAD

**Issue by xMinota** [cakebotpro/filebootstrap#1]
If you wish to submit cakebot to a bot list, you might need to exclude this feature as it's not allowed and you must make it opt-in or not included it at all.

**Comment by RDIL** [comment]
Can you provide a bit more detail on why this isn’t allowed?

**Comment by xMinota** [comment]
DBL refers to server lists in bots as intrusive to the servers'  privacy & etc., I'll provide a screenshot of this shortly.

**Comment by xMinota** [comment]
![Screenshot 2019-11-04 at 8 08 36 AM](https://user-images.githubusercontent.com/34592028/68123213-5a21c900-feda-11e9-8d54-4e7923208b6f.png)

**Comment by RDIL** [comment]
But this is limited to me

**Comment by xMinota** [comment]
Alright, it's good, but users can't use the command/it is still pretty much a breach of privacy anyways.

**Comment by RDIL** [comment]
As a privacy advocate, I see what you mean.  I’ll remove it in the December update.

**Comment by RDIL** [comment]
Is a count (integer value) allowed?

**Comment by xMinota** [comment]
Yes, 100%, you can do that, the only boundary is if you list the actual servers' names.

**Comment by RDIL** [comment]
Right.  This data is currently only accessible be me and our hosting provider.

**Comment by RDIL** [comment]
Reopening for proof that this has been resolved.
